### DISCUSSION TITLE
lseek problem

### DISCUSSION THREAD

**Discussion by WojciechGw** [picocomputer/community#134] [Q&A] [answered]
use lseek API function gives
Error: Call to undeclared function 'lseek'
any idea ?

CC65 app

**Comment by GlennFR** [comment]
Hi, can you be a little more specific ?  Exactly how are you compiling/linking your code (in the Visual Studio environment, directly using the cc65 suite...), and do the calls to open() or read() also flag the same errors?  Have you checked the spelling ("l" and not "1") ?  We can't provide help without more information.  lseek() works fine with cc65, the cc65 assembler, even my PLASMA interpreter that I'm writing.

**Reply by WojciechGw** [reply]
VS Code environment under Windows 10,
CC65 newest build within WSL2 Ubuntu
all other API functions are visible to VSCode and works
no typo errors 
open() read_xram() write_xram() without any issues
lseek is the only one problematic

PS. under LLVMOS lseek() works without problems

**Comment by rumbledethumps** [comment, marked as answer]
Maybe you forgot `#include <unistd.h>` which would make it undeclared.

**Reply by WojciechGw** [reply]
thank You

You've hit the nail on the head :)
